crates/containerd-shim-wasmtime/src/instance.rs

```diff
@@ -13,12 +15,22 @@ pub struct WasmtimeEngine {
     engine: wasmtime::Engine,
 }
 
+fn version_info() -> &'static RuntimeInfo {
+    static INFO: OnceLock<RuntimeInfo> = OnceLock::new();
+    INFO.get_or_init(|| RuntimeInfo {
+        name: "wasmtime",
+        version: option_env!("WASMTIME_VERSION_INFO").unwrap_or(env!("CARGO_PKG_VERSION")),
+    })
+}
```

Comment:
Is OnceLock necessary? 

```suggestion
static INFO: RuntimeInfo = RuntimeInfo {
    name: "wasmtime",
    version: option_env!("WASMTIME_VERSION_INFO").unwrap_or(env!("CARGO_PKG_VERSION")),
};

fn version_info() -> &'static RuntimeInfo {
    &INFO
}
```

Reply:
I will try this but I believe it is...

Reply:
it is:

> cannot call non-const fn `std::option::Option::<&str>::unwrap_or` in statics
calls in statics are limited to constant functions, tuple structs and tuple variants
consider wrapping this expression in `Lazy::new(|| ...)` from the `once_cell` crate: https://crates.io/crates/once_cellrustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B3%5D?3#file%3A%2F%2F%2Fhome%2Fjstur%2Fprojects%2Frunwasi%2Fcrates%2Fcontainerd-shim-wasmtime%2Fsrc%2Finstance.rs)